compiler/parser/src/context.rs

```diff
@@ -0,0 +1,173 @@
+use rustpython_ast::{Expr, ExprContext, ExprKind};
+
+pub fn set_context(expr: Expr, ctx: ExprContext) -> Expr {
```

Comment:
is this necessary part of end_location?

Reply:
Oh no, sorry, I need to revert this. Not sure how it got included.

Reply:
(Removed.)